pkg/solana/txm/pendingtx.go

```diff
@@ -68,9 +72,9 @@ type pendingTxContext struct {
 	cancelBy map[string]context.CancelFunc
 	sigToID  map[solana.Signature]string
 
-	broadcastedTxs      map[string]pendingTx  // transactions that require retry and bumping i.e broadcasted, processed
-	confirmedTxs        map[string]pendingTx  // transactions that require monitoring for re-org
-	finalizedErroredTxs map[string]finishedTx // finalized and errored transactions held onto for status
+	broadcastedProcessedTxs map[string]pendingTx  // broadcasted and processed transactions that may require retry and bumping
```

Comment:
Names except `Broadcasted` were chosen to be consistent with the [Solana Commitment Status](https://docs.anza.xyz/consensus/commitments/)

I feel it's good to reuse the same names across chains, but I also think `Processed` is clearer than `Unconfirmed` in this case. In my opinion, `Unconfirmed` does not make it clear whether the transaction was `Processed` or simply `Broadcasted`

@amit-momin @silaslenihan what do you think about `Broadcasted` vs `Unconfirmed`?

Reply:
I agree with Juan. `Unconfirmed` might be misleading in the context of Solana since we have this `Processed` state in between. In EVM, we could consider a broadcasted transaction as `Unconfirmed` since the next step in the lifecycle is `Confirmed`

Reply:
If Solana has these standard terms, then yes, better to align with that. I didn't know.